pkg/api/validation/validation.go

```diff
@@ -219,7 +219,10 @@ func ValidateObjectMetaUpdate(old, meta *api.ObjectMeta) errs.ValidationErrorLis
 	if len(meta.UID) == 0 {
 		meta.UID = old.UID
 	}
-	if meta.CreationTimestamp.IsZero() {
+	// ignore changes to timestamp
```

Comment:
Probably none.  I don't have the right "reset this value before ValidateUpdate is invoked" so the validator does the reset.

I suppose if no one ever set a creation timestamp (very old clusters), on an update having *a* value set is better than no value.